### ISSUE TITLE
Single Wildcard Game fails

### ISSUE THREAD

**Issue by erwstout** [erwstout/pinetar#9]
(JSON FILE LOGGED) 

A single WildCard game breaks Pinetar and nothing is displayed. Error received:

`Uncaught TypeError: games.map is not a function`

**Comment by erwstout** [comment]
Dump of wildcard JSON that fails:

```json
{ "subject": "master_scoreboard_mlb_2016_10_04", "copyright" : "Copyright 2016 MLB Advanced Media, L.P.  Use of any content on this page acknowledges agreement to the terms posted here http://gdx.mlb.com/components/copyright.txt", "data" : {"games":{"next_day_date":"2016-10-05","modified_date":"2016-10-04T20:52:43Z","month":"10","year":"2016","game":{"game_type":"F","double_header_sw":"N","location":"Toronto, Canada","away_time":"8:00","broadcast":{"away":{"tv":"TBS / SNET","radio":"105.7 The Fan, Orioles Radio Network, ESPN Radio"},"home":{"tv":"TBS / SNET","radio":"SN590, ESPN Radio"}},"time":"8:00","home_time":"8:00","home_team_name":"Blue Jays","description":"AL Wild Card Game","series_num":"1","original_date":"2016/10/04","home_team_city":"Toronto","venue_id":"14","gameday_sw":"P","away_win":"0","home_games_back_wildcard":"","away_probable_pitcher":{"last":"Tillman","stats_type":"R","name_display_roster":"Tillman","number":"30","era":"3.77","id":"501957","throwinghand":"RHP","s_losses":"6","first_name":"Chris","s_era":"3.77","stats_season":"2016","last_name":"Tillman","losses":"6","first":"Chris","s_wins":"16","wins":"16"},"away_team_id":"110","tz_hm_lg_gen":"ET","status":{"ind":"S","status":"Preview","inning_state":"","note":""},"home_loss":"0","home_games_back":"","home_code":"tor","away_sport_code":"mlb","home_win":"0","time_hm_lg":"8:00","ser_games":"1","away_name_abbrev":"BAL","league":"AA","if_necessary":"N","time_zone_aw_lg":"-4","away_games_back":"","home_file_code":"tor","game_data_directory":"/components/game/mlb/year_2016/month_10/day_04/gid_2016_10_04_balmlb_tormlb_1","time_zone":"ET","away_league_id":"103","home_team_id":"141","day":"TUE","time_aw_lg":"8:00","away_team_city":"Baltimore","tbd_flag":"N","tz_aw_lg_gen":"ET","away_code":"bal","series":"AL Wild Card","game_media":{"media":{"free":"NO","title":"BAL @ TOR","thumbnail":"http://mediadownloads.mlb.com/mlbam/preview/baltor_487595_th_7_preview.jpg","media_state":"media_on","tbs_auth_required":"Y","start":"2016-10-04T20:00:00-0400","has_mlbtv":"true","calendar_event_id":"14-487595-2016-10-04","enhanced":"N","type":"game"}},"game_nbr":"1","time_date_aw_lg":"2016/10/04 8:00","away_games_back_wildcard":"","scheduled_innings":"9","venue_w_chan_loc":"CAXX0504","first_pitch_et":"8:00","away_team_name":"Orioles","time_date_hm_lg":"2016/10/04 8:00","id":"2016/10/04/balmlb-tormlb-1","home_name_abbrev":"TOR","tiebreaker_sw":"N","ser_home_nbr":"1","ampm":"PM","home_division":"E","home_time_zone":"ET","away_time_zone":"ET","hm_lg_ampm":"PM","home_sport_code":"mlb","time_date":"2016/10/04 8:00","links":{"away_audio":"bam.media.launchPlayer({calendar_event_id:'14-487595-2016-10-04',gid:'2016_10_04_balmlb_tormlb_1',media_type:'audio'})","wrapup":"","preview":"/mlb/gameday/index.jsp?gid=2016_10_04_balmlb_tormlb_1&mode=preview&c_id=mlb","home_preview":"/mlb/gameday/index.jsp?gid=2016_10_04_balmlb_tormlb_1&mode=preview&c_id=mlb","away_preview":"/mlb/gameday/index.jsp?gid=2016_10_04_balmlb_tormlb_1&mode=preview&c_id=mlb","tv_station":"TBS / SNET","home_audio":"bam.media.launchPlayer({calendar_event_id:'14-487595-2016-10-04',gid:'2016_10_04_balmlb_tormlb_1',media_type:'audio'})","mlbtv":"bam.media.launchPlayer({calendar_event_id:'14-487595-2016-10-04',media_type:'video'})"},"home_ampm":"PM","game_pk":"487595","venue":"Rogers Centre","home_probable_pitcher":{"last":"Stroman","stats_type":"R","name_display_roster":"Stroman","number":"6","era":"4.37","id":"573186","throwinghand":"RHP","s_losses":"10","first_name":"Marcus","s_era":"4.37","stats_season":"2016","last_name":"Stroman","losses":"10","first":"Marcus","s_wins":"9","wins":"9"},"home_league_id":"103","video_thumbnail":"http://mediadownloads.mlb.com/mlbam/preview/baltor_487595_th_7_preview.jpg","away_loss":"0","resume_date":"","away_file_code":"bal","aw_lg_ampm":"PM","video_thumbnails":{"thumbnail":[{"content":"http://mediadownloads.mlb.com/mlbam/preview/baltor_487595_th_7_preview.jpg","height":"70","scenario":"7","width":"124"},{"content":"http://mediadownloads.mlb.com/mlbam/preview/baltor_487595_th_37_preview.jpg","height":"90","scenario":"37","width":"160"}]},"time_zone_hm_lg":"-4","away_ampm":"PM","gameday":"2016_10_04_balmlb_tormlb_1","away_division":"E"},"day":"04"}}}
```

It appears the objects change, by default pinetar is looking for `games: this.games.data.games.game` or something along those lines and it looks like this wildcard data drops the first `games`.

**Comment by erwstout** [comment]
I believe this is just an issue in general with single games....

**Comment by erwstout** [comment]
One game per day is now fixed in `1.0.0`
